account_lock_date_update/wizards/account_update_lock_date.xml

```diff
@@ -27,18 +24,16 @@
 
 
     <record model="ir.actions.act_window" id="account_update_lock_date_act_window">
-        <field name="name">Update accounting lock dates</field>
+        <field name="name">Lock Dates</field>
         <field name="res_model">account.update.lock_date</field>
         <field name="view_mode">form</field>
         <field name="target">new</field>
     </record>
 
-    <record model="ir.ui.menu" id="account_update_lock_date_menu">
-        <field name="name">Update accounting lock dates</field>
-        <field name="parent_id" ref="account.account_account_menu"/>
-        <field name="action" ref="account_update_lock_date_act_window"/>
-        <field name="groups_id" eval="[(6, 0, [ref('account.group_account_manager')])]"/>
-        <field name="sequence" eval="70"/>
-    </record>
+    <menuitem id="account_update_lock_date_menu"
+        parent="account.account_account_menu"
+        action="account_update_lock_date_act_window"
+        groups="account.group_account_manager"
+        sequence="70"/>
```

Comment:
I don't like to use thoses kind of shortcut for XML records because there is always some magic behind it. I prefer to declare everything and having the right field names. And I think there is no need to change it.

Reply:
@benwillig OK, i restored the initial XML code for that menu